### PR TITLE
Fix docker installed & add MIT license

### PR DESCRIPTION
**Pull request by torn4dom4n**
- Update links are broken
- Add MIT license

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ These instructions will cover usage information to help your run Flutter docker
 
 In order to run this image you'll need docker installed.
 
-* [Windows](https://docs.docker.com/windows/started)
-* [OS X](https://docs.docker.com/mac/started/)
-* [Linux](https://docs.docker.com/linux/started/)
+* [Mac OS](https://docs.docker.com/desktop/mac/install/)
+* [Linux](https://docs.docker.com/engine/install/#server)
+* [Windows](https://docs.docker.com/desktop/windows/install/)
```
Comment (stijnvdkolk) [inline README.md]: Any reason for the new ordering?

Comment (torn4dom4n) [reply README.md]: I made a mistaken. I'm going to use the alphabet to sort it. Sorry 😂.

**Review by stijnvdkolk — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -46,4 +46,4 @@ You can mount any volume you need to allow the image to scan its files.
 
 ## Copyright and license
 
-The MIT License (MIT) [http://www.opensource.org/licenses/mit-license.php](http://www.opensource.org/licenses/mit-license.php)
+License under the [MIT](LICENSE) license.
```
Comment (Meldiron) [inline README.md]:
```suggestion
The MIT License (MIT) [http://www.opensource.org/licenses/mit-license.php](http://www.opensource.org/licenses/mit-license.php)
```

**Review by Meldiron — CHANGES_REQUESTED** [review]
Requested tiny change

**Review by christyjacob4 — APPROVED** [review]

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+is%3Aissue+label%3Ahacktoberfest+created%3A%3E2021-01-01+is%3Aopen&type=issues) and help make Appwrite better :)

**Pull request merged**